### PR TITLE
Update v1.3 documentation

### LINKED ISSUE #80: Write Preliminary Documentation: User Guide & Developer Guide
TAHub has undergone several changes since the MVP implementation, notably the addition of support for multiple types of arguments and changes to searching behavior. 

As such, updates to the User Guide & Developer Guide will need to be made.

### PR DESCRIPTION
**Pull request by S-K-Y-Light**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -129,23 +133,23 @@ Examples:
...
-* Can delete multiple items at once
+* Can delete multiple students at once.
 
 Examples:
-* `list` followed by `delete i/2` deletes the 2nd student in the address book.
-* `list` followed by `delete i/2 i/3` deletes the 2nd and 3rd student in the address book.
-* `find Betsy` followed by `delete i/1` deletes the 1st student in the results of the `find` command.
+* `list` followed by `delete 2` deletes the 2nd student in TAHub.
+* `list` followed by `delete 2,3` deletes the 2nd and 3rd student in TAHub.
+* `find n/Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
```
Comment (S-K-Y-Light) [inline docs/UserGuide.md:149]: TODO: Confirm with Team that Find & Delete Feature interact together as per above

Comment (S-K-Y-Light) [reply docs/UserGuide.md:149]: Confirmed that feature interacts together as per above

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,9 +17,9 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F13-1/tp/releases).
```
Comment (notnotmax) [inline docs/UserGuide.md]: Everything shows up as step 1 on Github. Is this intentional or a display error?

**Review by notnotmax — COMMENTED** [review]
Overall lgtm except a minor nit. See comment for details.

File: docs/UserGuide.md (hunk 2 above)
Comment (S-K-Y-Light) [reply docs/UserGuide.md]: Good Catch. Made a Push to change the Indices in Quick Start

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -461,20 +461,61 @@ testers are expected to do more *exploratory* testing.
...
+   Expected: Student `TestOne` is added to the list. Details of the added student is shown.
+   2. Test case: `add n/TestOne p/11111111`<br>
+   Expected: No student is added. Error details shown.
+   3. Test case: `add n/TestOne e/test1@example.com c/CS2103T`<br>
+   Expected: No student is added. Error details shown.
+   4. Test case: `add n/Test1 p/11111111 e/test1@example.com c/CS2103T`<br>
+      Expected: No student is added. Error details shown.
 
+### Finding a student (by course)
+1. Finding a student (by course)
```
Comment (notnotmax) [inline docs/DeveloperGuide.md]: Could this be changed to just be the header (with ###), e.g. remove line 477?
Same for the other testcases around this one.

Comment (S-K-Y-Light) [reply docs/DeveloperGuide.md]: Sure. Updated as requested.

**Pull request merged**